samples/todolist.md

```diff
@@ -0,0 +1,160 @@
+# Pelorus Samples & Use Cases: Todolist
+
+## Deploying a todolist Sample alongside the Webhook/Generic CommitTime Exporter
+
+* The Pelorus exporter requires the Pelorus git source and branch
+  * Fork [this repo](https://github.com/konveyor/pelorus) to https://github.com/your_org/pelorus
+* We'll need a sample application to measure the commits and deployments
+  *  Fork [this repo](https://github.com/konveyor/mig-demo-apps) to https://github.com/your_org/mig-demo-apps
+  * Note: We'll be using the todolist-mongo-go application
+* The definitive Pelorus install guide can be found [here](https://pelorus.readthedocs.io/en/stable/Install/)
+
+## Pelorus install steps
+1. Create the pelorus namespace
+```
+oc create namespace pelorus
+```
+2. Install the required granfa and prometheus operators
+```
+helm install operators charts/operators --namespace pelorus
+```
+3. Create a GitHub Personal Access Token and store it as a secret in Openshift:
+```
+oc create secret generic github-secret --from-literal=GIT_USER=<username> --from-literal=GIT_TOKEN=<personal access token> --namespace pelorus
+```
+4. Install Pelorus
+```
+helm install pelorus charts/pelorus --namespace pelorus
+```
+5. Check the install
+```
+oc get all -n pelorus
+```
+***Note*** pause to allow the install to fully complete.
+
+## Pelorus configuration
+1. Make a copy of the values.yaml file [provided in this repo](https://github.com/konveyor/pelorus/blob/master/charts/pelorus/values.yaml) and save it to /var/tmp/values.yaml
+```
+cp charts/pelorus/values.yaml /var/tmp/
+```
+2. Update the /var/tmp/values.yaml file with the following example:
+```
+exporters:
+  instances:
+  - app_name: deploytime-exporter
+    source_context_dir: exporters/
+    extraEnv:
+    - name: APP_FILE
+      value: deploytime/app.py
+    - name: LOG_LEVEL
+      value: DEBUG
+    - name: NAMESPACES
+      value: mongo-persistent
+    source_ref: master
+    source_url: https://github.com/your_org/pelorus.git
+  - app_name: committime-generic
```

Comment:
What does generic mean here?

Reply:
I believe there are several flavors of the exporter for git..  github, generic, gitlab I think.  I think I also pulled this from the original doc